env/work/srw_python/basic-test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
```

Comment:
Test script from @robnagler. Should be added along with the updated Makefile.

Reply:
OK as a temporary solution. On a long run, I would strongly prefer this to be a cross-platform Python script, with more sophisticated error checking, etc. I think we have already a ticket for this (?)

Reply:
Yes, please see https://github.com/radiasoft/SRW-light/issues/5.

Reply:
I also changed this bash file name to "srwl_test_basic.sh", and made the corresponding correction in Makefile.